### PR TITLE
Add registry parameter to k8s

### PR DESCRIPTION
**Pull request by wlan0**
@alena1108

### CODE REVIEW

**Comment by alena1108** [comment]
@wlan0 new folder needs to be created as we haven't introduced new template for 1.5.0 yet. Also k8s image needs to be changed given corresponding package changes https://github.com/rancher/kubernetes-package/pull/27

**Comment by deniseschannon** [comment]
@alena1108 we have made a new version for 1.5.0. it's folder 19, so i think this could be merged?

**Comment by alena1108** [comment]
@deniseschannon if the new folder was created, we still need a new k8s image as @wlan0 changed kubernetes-package, and we have to build a new image

File: infra-templates/k8s/19/rancher-compose.yml [hunk 1]
```diff
@@ -46,6 +46,12 @@
     required: false
     default: "gcr.io/google_containers/pause-amd64:3.0"
     type: string
+  - variable: REGISTRY
+    description: The registry to pull addon images from. This should be set to the private registry from which the images for helm, heapster, dashboard and dns (and other addon-services) should be pulled.
+    label: Registry
+    required: false
+    default: "docker.io"
```
Comment (alena1108) [inline infra-templates/k8s/19/rancher-compose.yml]: Should be none as there is gcr.io and docker.io registries used in kubernetes-package

**Pull request merged**